lpsgateway/lpsgateway.ino

```diff
@@ -0,0 +1,70 @@
+
+// Including  RadioHead library 
+#include <SPI.h>
+#include <RH_RF95.h>
+#include <RHReliableDatagram.h>
+
+#define BAUDRATE 115200
+#define BONJOUR "LAUNCHPADSTATION"
+#define RFM95_CS 10
+#define RFM95_RST 9
+#define RFM95_INT 2
+#define RF95_FREQ 915.0
+
+RH_RF95 rf95(RFM95_CS, RFM95_INT);
+
+void setup()
+{  
+  init_communication();
+}
+
+void loop()
+{
+  if (Serial.available() > 0)
+  {
+    uint8_t radiopacket = Serial.read();
+    
+    delay(10);
+    rf95.send(&radiopacket, 1);
+    rf95.waitPacketSent();
+
+    delay(10);
+    if (rf95.waitAvailableTimeout(1000))
+    {  
+      uint8_t buf[RH_RF95_MAX_MESSAGE_LEN];
+      uint8_t len = sizeof(buf);
+      if (rf95.recv(buf, &len))
+      {
+        Serial.write(*buf);Serial.write('\r');Serial.write('\n');
+      }
+    }
```

Comment:
There's no risk the delay to receive status is longer than 1 s after we sent a command to the LPS? In that case we'd just read the new status after we sent another command, right?